app/ui-react/packages/ui/src/Integration/IntegrationDetailBreadcrumb.tsx

```diff
@@ -0,0 +1,83 @@
+import * as H from 'history';
+import { DropdownKebab } from 'patternfly-react';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { Breadcrumb, ButtonLink } from '../Layout';
+
+interface IBreadcrumbMenuActions {
```

Comment:
I don't think we need specific menu action interfaces for this breadcumb component *and* the history list view component that comes after it.  Since this is kinda generic maybe it'd be worth having a `models.ts` file in `Shared` that has this kind of thing that could be shared instead.

Reply:
You got it, makes sense. 👍